scripts/extract-paired-reads.py

```diff
@@ -31,15 +31,29 @@
 
 def get_parser():
     epilog = """
-    The output is two files, <input file>.pe and <input file>.se, placed in the
-    current directory. The .pe file contains interleaved and properly paired
-    sequences, while the .se file contains orphan sequences.
-
-    Many assemblers (e.g. Velvet) require that you give them either perfectly
-    interleaved files, or files containing only single reads. This script takes
-    files that were originally interleaved but where reads may have been
-    orphaned via error filtering, application of abundance filtering, digital
-    normalization in non-paired mode, or partitioning.
+    Separate properly interleaved reads from orphaned reads.
```

Comment:
Any reason to repeat (though with different wording) the script summary at https://github.com/dib-lab/khmer/pull/1085/files#diff-7ba8eb069734923ae2a8b85fa4021e7eR63 here?